app/serializers/event_serializer.rb

```diff
@@ -11,7 +11,7 @@ class EventSerializer < ActiveModel::Serializer
   has_one :vote, serializer: VoteSerializer
 
   def actor
-    object.eventable.try(:user)
+    object.user || object.eventable.try(:user)
```

Comment:
Erg, the wonkiness of the events system is really killing us here; why do we need to set the discussion_id on motion events? Why is the actor sometimes on the event directly and sometimes on the eventable? Why is event.discussion ok sometimes, and Event.where(discussion_id: eventable.discussion_id) necessary in others?

I think streamlining this is probably we need to take a look at sometime in the medium-term future, since it's such a central part of the app (and will become more central as we move towards plugin architecture / app-to-app comms)